### PR TITLE
add a section of "How to cite" Adaptive

### PR DESCRIPTION
**Pull request by basnijholt**
I frequently get this question, so I think it will be good to add it to the website.

This will probably be good until we have published [the paper](https://gitlab.kwant-project.org/qt/adaptive-paper).

I've made Adaptive italic because this is what is recommended (in a writing course that I took) for names of software packages.

The DOI I chose will always point to the latest release on Zenodo.

@akhmerov and @jbweston, what do you think?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -121,6 +121,22 @@ We implement several other checks in order to maintain a consistent code style.
 
 in the repository.
 
+How to cite
+-----------
+
+If you used Adaptive in the preparation of a manuscript, we would appreciate it if you cite us.
```
Comment (akhmerov) [inline README.rst]: Perhaps `If you used Adaptive in a scientific work, please cite it as follows.`

File: README.rst [hunk 2]
```diff
@@ -121,6 +121,22 @@ We implement several other checks in order to maintain a consistent code style.
 
 in the repository.
 
+How to cite
```
Comment (akhmerov) [inline README.rst]: Citing

File: README.rst (hunk 1 above)
Comment (basnijholt) [reply README.rst]: Fixed.

File: README.rst [hunk 3]
```diff
@@ -121,6 +121,21 @@ We implement several other checks in order to maintain a consistent code style.
...
+------
+
+If you used Adaptive in a scientific work, please cite it as follows.
+
+.. code::
+
+    @misc{Nijholt2019,
+      doi = {10.5281/zenodo.1182437},
+      author = {Bas Nijholt and Joseph Weston and Jorn Hoofwijk and Anton Akhmerov},
+      title = {\textit{Adaptive}: Tools for adaptively sampling of mathematical functions in parallel},
```
Comment (basnijholt) [inline README.rst]: I've slightly rephrased it. I think it is better because otherwise `adaptive` appears twice in close procession.

Comment (jbweston) [inline README.rst]: Change 
```
Tools for adaptively sampling of mathematical functions in parallel
```
to
```
Tools for parallel adaptive sampling of mathematical functions
```

Comment (jbweston) [reply README.rst]: minimally we could just correct the english to
```
Tools for adaptively sampling mathematical functions in parallel
```

**Pull request merged**